.github/workflows/build.yml

```
@@ -1,6 +1,10 @@
 name: Python Fire
 
-on: [push]
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
```

This is unfriendly to contribution.

People should contribute using [feature branches](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches), and we should allow people to test their work on their fork before opening a PR.

```suggestion
on: [push, pull_request]
```